Add short description of how to configure Tor. (#464)
comit-network/xmr-btc-swap: docs/asb/README.md

Note that currently there is no specific error sent back to the CLI for such kind of cases, so a user might not know why the swap execution was rejected.
Note that there is currently no notification service implemented for low funds.
The ASB provider has to monitor Monero funds to make sure the ASB still has liquidity.

#### Tor and hidden services

> The ASB supports will automatically create a Tor hidden service if the Tor control port can be found.

Reviewer: ```suggestion
The ASB supports Tor and will automatically create a Tor hidden service if the Tor control port can be found.
```

By default, the ASB will look for the control port under `localhost:9051`.

> To allow the ASB to create hidden services, enable the control port and authentication in your torrc file.

Reviewer: ```suggestion
To allow the ASB to create a hidden service, enable the control port and authentication in your torrc file:
```

> Concretely, add these lines:

Reviewer: ```suggestion
```

```
ControlPort 9051
CookieAuthentication 1
CookieAuthFileGroupReadable 1
```

It is important that the user running the ASB has the correct user rights, i.e. is in the same group as the user running Tor.
E.g. if running on debian and having Tor install via apt, add your user to the following group:
`sudo adduser $(whoami) debian-tor`.

> When configured correctly, your ASB will print the created onion addresses:

Reviewer: ```suggestion
When configured correctly, your ASB will print the created onion addresses upon startup:
```
